src/pages/learn.js

```diff
@@ -57,17 +57,33 @@ const callStitch = async (metadata, key, callback) => {
     const res = await callStitchFunction('fetchDevhubMetadata', metadata, key);
     callback(parseArticles(res));
 };
-
+// strip out the 'All' param from the url and the stitch function key
+const stripAllParam = filterValue => {
+    Object.keys(filterValue).map(key => {
+        if (filterValue[key] === 'all') {
+            delete filterValue[key];
+        }
+    });
+    return filterValue;
```

Comment:
nit: maybe I'm misinterpreting this but it seems we modify `filterValue` in place so the references to it should be updated without the need to return anything, but maybe there is some value in passing another reference?

Reply:
yeah this function can definitely be cleaned up! thanks!